### PR TITLE
Show all users for lockbox

### PR DESCRIPTION
**Pull request by BintLopez**
## Changelog
- adds display of users for a given lockbox

## Link to issue:  
https://github.com/MidwestAccessCoalition/lockbox_rails/issues/191

## Steps for QA/Special Notes:
- N/A

## Relevant Screenshots:  
image

## Are you ready for review?:

- [x] Added relevant tests
- [x] Linked PR to the issue
- [x] Added notes for QA/special notes
- [x] Added revelant screenshots

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -27,6 +27,18 @@ def has_signed_in?
     !!last_sign_in_at
   end
 
+  def status
+    return "pending" if confirmed_at.nil?
+    return "locked" if locked_at.present?
+    return "active"
+  end
+
+  def available_action_text
```
Comment (BintLopez) [inline app/models/user.rb]: add a test for this

File: app/controllers/lockbox_partners/users_controller.rb [hunk 2]
```diff
@@ -12,13 +12,44 @@ def create
...
+      locked_status = @user.locked_at.nil? ? 'unlocked' : 'locked'
+      flash[:notice] = "User account for #{@user.email} has been #{locked_status}."
+      redirect_to lockbox_partner_users_path
+    else
+      flash[:alert] = @user.errors.full_messages.join(', ')
+      redirect_to lockbox_partner_users_path
+    end
+  end
+
+  def resend_invite
```
Comment (BintLopez) [inline app/controllers/lockbox_partners/users_controller.rb]: add a controller test here

File: app/controllers/lockbox_partners/users_controller.rb [hunk 3]
```diff
@@ -12,13 +12,44 @@ def create
...
+      render :index
     end
   end
 
+  def index
+    @new_user = @lockbox_partner.users.new
+    @users = @lockbox_partner.users.all
+  end
+
+  def update
```
Comment (BintLopez) [inline app/controllers/lockbox_partners/users_controller.rb]: add a controller test here

File: config/routes.rb [hunk 4]
```diff
@@ -15,7 +15,9 @@
 
   resources :lockbox_partners, only: [:new, :create, :show, :edit, :update] do
     scope module: 'lockbox_partners' do
-      resources :users, only: [:new, :create]
+      resources :users, only: [:new, :create, :index, :update] do
+        get 'resend_invite', to: 'users#resend_invite'
```
Comment (kyaroch) [inline config/routes.rb]: I think this should be a POST or PATCH request. Making it a GET could lead to invites being resent inadvertently.

**Comment by BintLopez** [comment]
image

File: config/routes.rb (hunk 4 above)
Comment (BintLopez) [reply config/routes.rb]: done

**Pull request merged**